integration/samples/package-json/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "@sample/package-json",
+  "description": "A sample library with configuration in package.json",
+  "version": "1.0.0-pre.0",
+  "private": true,
+  "repository": "https://github.com/dherges/ng-packagr.git",
+  "peerDependencies": {
+    "@angular/core": "^4.1.2",
+    "@angular/common": "^4.1.2"
+  },
+  "ngPackage": {
+    "$schema": "../../../src/lib/ng-package.schema.json",
```

Comment:
Ah, do you know any possibility to extend the default schema of a `package.json`? 

Here is one ... quite old from 2013: https://www.npmjs.com/package/package.json-schema

Which schema is used for example by vscode, atom, ...? Could think of extending one of those ...

Reply:
Here is one: http://json.schemastore.org/package

Reply:
Got it!